.github/workflows/build-and-deploy.yml

```diff
@@ -209,8 +211,10 @@ jobs:
             printf '#!/bin/sh\n\nexec /mingw64/bin/git.exe "$@"\n' >/usr/bin/git
           } &&
 
+          MINGW_ARCHS_TO_BUILD=$(test "$ARCHITECTURE" == "aarch64" && echo "clangarm64" || echo "mingw32 mingw64")
```

Comment:
@dennisameling I hope you don't mind that I edited this: We can either use a Bash-style ternary (with `$ARCHITECTURE`) or use GitHub workflow syntax (`${{ env.ARCHITECTURE == 'aarch64' && 'clangarm64' || 'mingw32 mingw64' }}`), but we should not mix them. I opted for the former.

Reply:
Works for me! 👍🏼